master/internal/command/tensorboard_manager.go

```diff
@@ -200,7 +200,7 @@ func (t *tensorboardManager) newTensorBoard(
 				RawHostPath:      c.HostPath(),
 				RawPropagation:   ptrs.StringPtr(model.DefaultSharedFSPropagation),
 			}).(expconf.BindMount)
-			uniqMounts[sharedFSMount] = true
+			uniqMounts[sharedFSMount.RawContainerPath] = model.ToModelBindMount(sharedFSMount)
```

Comment:
Can you use `.ContainerPath()` instead/  The `Raw*` API should only be used when you are first parsing an object.

Reply:
good point. changed.